beta/src/components/header/index.ts

```diff
@@ -433,14 +433,16 @@ class Header extends Component<HeaderProps> {
   }
 
   renderSearch (): HTMLElement {
+    const searchClasses: string = 'search flex-l flex-auto-l w-100-l justify-center-l'
     const search = {
       on: () => this.state.cache(Search, 'search').render({ tags: TAGS }),
       off: () => {
         const attrs = {
```

Comment:
Unrelated to this PR, but this way of adding attrs to a button is quite odd imo. Better to just put them straight in the HTML (could always put them over multiple lines etc.) rather than add more complexity.

Reply:
Good call, addressed here: https://github.com/resonatecoop/stream/pull/216/commits/45c7e7cf4dffd8596f586d8d6ee5befa740052aa